pyg_lib/ops/__init__.py

```diff
@@ -6,7 +6,8 @@
 from .scatter_reduce import fused_scatter_reduce
 
 
-def grouped_matmul(inputs: List[Tensor], others: List[Tensor]) -> List[Tensor]:
+def grouped_matmul(inputs: List[Tensor], others: List[Tensor],
```

Comment:
I think I would like to keep `grouped_matmul` and `segment_matmul` interfaces consistent. We can either add this functionality to `segment_matmul` as well or solely perform adding the bias on PyG side (my current preference).